crates/extension-catalog/src/load.rs

```diff
@@ -47,6 +47,11 @@ mod tests {
             sdk_version: "0.3.0".parse().unwrap(),
             minimum_gateway_version: "0.90.0".parse().unwrap(),
             sdl: Some("directive foo on SCHEMA".to_string()),
+            license: None,
```

Comment:
Could we also have these in the init and build commands already?

Reply:
Done